docs/xls-70d-on-chain-credentials/index.md

```diff
@@ -0,0 +1,47 @@
+# On-Chain Credentials
+
+The On-Chain Credentials feature is a set of tools for managing authorization and compliance requirements using the XRP Ledger blockchain, while respecting privacy and decentralization. This feature extends and interconnects with other features of the XRP Ledger including [Decentralized Identifiers (DID)](https://xrpl.org/docs/concepts/accounts/decentralized-identifiers) and [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth). The goal of this feature is to streamline the process of compliance checks such as [KYC (Know Your Customer)](https://en.wikipedia.org/wiki/Know_your_customer) requirements and to enable further trust-based applications within the XRP Ledger ecosystem.
+
+The design of On-Chain Credentials draws from the [W3C Verifiable Credentials standard](https://www.w3.org/TR/vc-data-model-2.0/) and is intended to be compatible to an extent that makes sense in the context of the XRP Ledger. There are some differences in data structures and formatting: for example, the subject of a credential is identified by an XRP Ledger address rather than a URL.
+
+## Overview
+
+_Credentials_ are signed statements that can be stored in the ledger (in other words, on-chain) and can attest to a user's identity, legal status, or other status. This feature includes issuance, storage, and verification of credentials directly on the XRP Ledger, while still supporting the privacy needs of users.
+
+This feature set is designed for an ecosystem of parties with the following roles:
+
+- _Authorizers_ who want to limit specific actions or interactions to users who possess the right credentials.
+- _Issuers_ who give out credentials to users who meet their criteria.
+- _Users_ who obtain credentials from issuers and use those credentials to interact with authorizers.
+
+Each credential applies to a specific user's XRP Ledger account, and is attested by a specific credential issuer. The exact qualifications that a credential entails are open-ended, and defined by their issuers. They can range from trivial to serious, or broad to narrow. Some examples of things a credential could state:
```

Comment:
> and is attested by a specific credential issuer

It's unclear what this part of the sentence is trying to say -- I lean toward simply removing it. In my current thinking, an issuance of a credential is subtly different from an attestation -- though perhaps I could be wrong. Are there times where a credential isn't attenting anything?

What about this?

```suggestion
Each credential applies to a specific user's XRP Ledger account. The exact qualifications of any particular credential are open-ended, and defined by each issuer. Credential meaning can range from trivial to serious, or broad to narrow. Some examples of things a credential could state:
```

Reply:
I disagree on this. A credential is a signed statement by the issuer, which is IMO exactly what an _attestation_ is: a signed statement by a known party.

Reply:
Good point -- re-reading this with fresh eyes, I agree with you and prefer the original wording. I'll resolve this one.